include/wx/msw/listctrl.h

```diff
@@ -184,7 +187,7 @@ class WXDLLIMPEXP_CORE wxListCtrl: public wxListCtrlBase
     bool SetItemData(long item, long data) { return SetItemPtrData(item, data); }
 
     // Gets the item rectangle
-    bool GetItemRect(long item, wxRect& rect, int code = wxLIST_RECT_BOUNDS) const;
+    virtual bool GetItemRect(long item, wxRect& rect, int code = wxLIST_RECT_BOUNDS) const;
```

Comment:
```suggestion
    bool GetItemRect(long item, wxRect& rect, int code = wxLIST_RECT_BOUNDS) const wxOVERRIDE;
```

Also in generic header.

Reply:
@MaartenBent ,
Look at the ticket on trac.
Unless you have some strong objection to Vadim' (last) comment, I'd leave it as is.

Reply:
The last comment suggests to add a pure virtual function to the base class. This is what you have done in `listbase.h`. This virtual function needs to be overridden in all classes that extend the base class, in this case `msw/listctrl.h` and `generic/listctrl.h`.

There is no need to add `virtual` again, because this is inherited from the base class.

`wxOVERRIDE` is used to let the compiler know that you intend to override a function. If that function gets renamed or removed in the base class, compilation will fail because there is nothing to override. 

Note that `wxOVERRIDE` does nothing in vs2010 so you might not see its effect.

Reply:
OK, fixing

Reply:
Just a note: it's really unnecessary to reply "fixing" to every comment, this isn't useful and just results in dozens of trivial emails for every PR. Please just fix them if there is nothing more to say. TIA!

Reply:
@MaartenBent ,
thank you for clearing up the difference between "virtual" and "override".